### ISSUE TITLE
Grep for tabs is actually grep for "t"

### ISSUE THREAD

**Issue by MrOlm** [AstrobioMike/GToTree#23]
Hello!

I was running GToTree and getting the following error:

```
$ GToTree -A small_faa.txt -H Universal -j 10 -o GammaTree_v1


                                  GToTree v1.5.34
                         (github.com/AstrobioMike/GToTree)


  small_faa.txt has tabs in it, maybe they are at the end of all of the entries?
  They might cause problems. Check it out and provide one with no tabs.

Exiting for now.
```

My file does not have tabs in it, so I took a quick look at the source code and saw this this check is being done with the line ```grep "\t" small_faa.txt```. Running that on my file highlights all of the locations in my paths that have lowercase "t"s. When I modified my file to remove these lowercase "t"s, the program started working.

Changing the grep command to ```grep $'\t' small_faa.txt``` showed no hits in my file, and I believe that this is the correct way to grep for tabs and other special characters. I'm running ```grep (GNU grep) 3.4```

I'm able to work around having lowercase "t"s in my input, so no rush to fix on my end, but just wanted to let you know. Thanks for writing this tool!

-Matt

**Comment by AstrobioMike** [comment]
Ah, it’s definitely more portable doing it with `$'\t'`, amateur hour! And man that would be super annoying as a user (fortunately I only poorly added that check very recently after debugging a user snag).

Thanks so much for the note, Matt! Greatly appreciated!

**Comment by AstrobioMike** [comment]
Fixed and conda-ready as of v1.5.35! 

I definitely fell victim to a careless BSD-grep quick-testing when i implemented that :/  

Thank you again for taking the time to look and let me know :)

**Comment by MrOlm** [comment]
Great, thanks for the quick fix!
